tests/Io/FileSystemTest.php

```
@@ -42,6 +42,132 @@ public function tearDown()
+
+        $this->assertSame('data1' . PHP_EOL . 'data2', FileSystem::read('./temp_file'));
+    }
+
+    public function testWriteWithLockFlag()
+    {
+        $this->assertSame(0, FileSystem::write('./temp_lock_file', '', false, true));
+    }
+
+    public function testReadWithMaxLength()
+    {
+        FileSystem::write('./temp_lock_file', 'data');
+
+        $this->assertSame('dat', FileSystem::read('./temp_lock_file', 0, 3));
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error copying ./temp_lock_file to /temp_lock_file
+     */
+    public function testCopyShouldThrowIOException()
+    {
+        FileSystem::write('./temp_lock_file', 'data');        
+        FileSystem::copy('./temp_lock_file', '/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error moving ./temp_lock_file to /temp_lock_file
+     */
+    public function testMoveShouldReturnIOException()
+    {
+        FileSystem::write('./temp_lock_file', 'data');        
+        FileSystem::move('./temp_lock_file', '/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error deleting /temp_lock_file
+     */
+    public function testDeleteShouldReturnIOException()
+    {
+        FileSystem::delete('/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error creating directory /temp_directory
+     */
+    public function testCreateDirectoryShouldReturnIOException()
+    {
+        FileSystem::createDirectory('/temp_directory');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error creating directories /temp_directory
+     */
+    public function testCreateDirectoriesShouldReturnIOException()
```

Should be called `Throw` instead of `Return`